.github/workflows/pr-screenshots.yml

```diff
@@ -0,0 +1,46 @@
+---
+name: Automatically create and attach screenshots for PRs that affect dashboards
+
+on:
+    pull_request:
+        branches:
+            - main
+        paths:
+            - microlith/grafana/provisioning/dashboards/**
+
+    workflow_dispatch:
+        inputs:
+            ref:
+                description: Branch, tag, or commit SHA to run against
+                required: false
+                default: main
```

Comment:
Looks like you're defining `$ref` but not using it anywhere?

Reply:
Ah yeah, wanted to add that to checkout